packages/ui-library/src/components/forms/toggle-switch/index.stories.ts

```diff
@@ -4,76 +4,463 @@ import { BlrToggleSwitchRenderFunction } from './renderFunction';
+      table: {
+        category: 'Content / Settings',
+      },
+      if: { arg: 'showStateLabel', eq: true },
+    },
+    onLabel: {
+      description: 'Enter string used as on label text.',
+      table: {
+        category: 'Content / Settings',
+      },
+      if: { arg: 'variant', eq: 'leading' },
+    },
+    offLabel: {
+      description: 'Enter string used as off label text.',
+      table: {
+        category: 'Content / Settings',
+      },
+      if: { arg: 'variant', eq: 'leading' },
+    },
+    disabled: {
+      description:
+        'Choose if component is disabled. Prevents the user to select or change the value of this component.',
+      table: {
+        category: 'States',
+      },
+    },
+    checkInputId: {
+      description: 'Unique identifier for this component.',
+      table: {
+        category: 'Technical attributes',
+      },
+    },
+    name: {
+      description: 'For a < form > element, the name attribute is used as a reference when the data is submitted.',
+      table: {
+        category: 'Technical attributes',
+      },
+    },
+    onChange: {
+      description: 'Fires when the value changes.',
+      table: {
+        category: 'Events',
+      },
+    },
+    onFocus: {
+      description: 'Fires when the component is focused.',
+      table: {
+        category: 'Events',
+      },
+    },
+    onBlur: {
+      description: 'Fires when the component lost focus.',
+      table: {
+        category: 'Events',
+      },
+    },
+    readonly: {
+      description: 'Choose if component is readonly. The user can select but not change the value of this component.',
+      table: {
+        category: 'States',
+      },
     },
   },
   parameters: {
     design: {
       type: 'figma',
       url: 'https://www.figma.com/file/C4vgEKz8mKyulJ4gm3Qdql/%F0%9F%AB%A7-%5BBLR%5D-The-B01LER?node-id=3618%3A125201&mode=dev',
     },
+    layout: 'centered',
+    docs: {
+      description: {
+        component: `<Markdown>
+Toggle Switch allows users to choose between two mutually exclusive states, such as on/off or yes/no. It is typically a small, rectangular button with two states, represented by different icons or labels.
+They can also be used to control more complex features, such as the state of a system or the availability of a service.
+- [**Appearance**](#appearance)
+ - [**Size Variant**](#size-variant) 
+ - [**Has State Label**](#has-state-label) 
+- [**Content / Settings**](#content--settings)
+ - [**Checked**](#checked) 
+ - [**On Label**](#on-label) 
+ - [**Off Label**](#off-label)  
+- [**States**](#states)
+ - [**Disabled**](#disabled) 
+ - [**Readonly**](#readonly)
+- [**Dependencies**](#dependencies)
+ - [**Form Caption**](#form-caption)  
+</Markdown>
+        `,
+      },
+    },
   },
 };
 
 export const BlrToggleSwitch = (params: BlrToggleSwitchType) => BlrToggleSwitchRenderFunction(params);
 
 BlrToggleSwitch.storyName = 'ToggleSwitch';
+BlrToggleSwitch.args = defaultParams;
```

Comment:
```suggestion
export const ToggleSwitch = (params: BlrToggleSwitchType) => BlrToggleSwitchRenderFunction(params);
ToggleSwitch.args = defaultParams;
```

Reply:
We can just use the more understandable naming version as the component declaration.

Reply:
done